CMakeLists.txt

```diff
@@ -10,7 +10,7 @@
 # CMake versions greater than the ITK_NEWEST_VALIDATED_POLICIES_VERSION policies will
 # continue to generate policy warnings "CMake Warning (dev)...Policy CMP0XXX is not set:"
 #
-set(ITK_OLDEST_VALIDATED_POLICIES_VERSION "3.16.3")
+set(ITK_OLDEST_VALIDATED_POLICIES_VERSION "3.18.0")
 set(ITK_NEWEST_VALIDATED_POLICIES_VERSION "3.29.0")
```

Comment:
FYI: homebrew mac version of 3.31.2 seems to work fine too.

Reply:
We can update the highest verified version in a separate PR. That update needs some testing.